### PR TITLE
add del to parallel set

### PR DESCRIPTION
**Pull request by stoeffel**
As discussed in #65

### CODE REVIEW

**Comment by tomekwi** [comment]
:+1: big commit :)

**Comment by stoeffel** [comment]
@tomekwi thanks man :smile_cat:

**Comment by stoeffel** [comment]
:bell:

**Comment by tjmehta** [comment]
Checking this out now.

File: README.md [hunk 1]
```diff
@@ -131,6 +131,36 @@ function join() { return Array.prototype.slice.call(arguments).join(''); }
 curry(join, 3)(1)(0)(1); // "101"
 ```
 
+## del
+
+Functional version of delete obj[key] which returns the same obj without the deleted key.
```
Comment (tjmehta) [inline README.md]: Add a note: supports keypaths by default

**Comment by tjmehta** [comment]
LGTM, waiting on readme tweak about keypaths

File: README.md (hunk 1 above)
Comment (stoeffel) [inline README.md]: Will add the note as soon as I'm on a labtop. I'm in military this month (mandatory in Switzerland) so it's kind of hard :smile: .

Comment (tjmehta) [inline README.md]: Ill add it for you :)


—
@tjmehta

Comment (stoeffel) [inline README.md]: Thanks :+1:

**Comment by tjmehta** [comment]
Oh well didn't realize your example utilizes the key path functionality. I think that is sufficient for now

**Comment by tjmehta** [comment]
101@0.13.0

**Pull request merged**